manuscripts/esquery.py

```diff
@@ -29,29 +29,26 @@
 
 from elasticsearch_dsl import A, Search, Q
 
-USE_ELASTIC_DSL = True
-
 
 class ElasticQuery():
     """ Helper class for building Elastic queries """
 
     AGGREGATION_ID = 1  # min aggregation identifier
     AGG_SIZE = 100  # Default max number of buckets
-    ES_PRECISION = 3000
+    ES_PRECISION = 3000  # This is the default value for percision_threshold
 
     @classmethod
     def __get_query_filters(cls, filters=None, inverse=False):
```

Comment:
I would use "filters=[]". Since it is expected to be a list, this way you don't need, in the implementation, to check for None, and it is even more explicit that you're expecting a list for that argument. I would only use None if you find a case where you need to make a difference between that "None" and an empty list as an argument.